src/hooks/kakaoMap/useRecommendRandomRestaurant.ts

```diff
@@ -70,7 +70,7 @@ const useRecommendRandomRestaurant = () => {
               height: 96px;
             ">
             <img
-            class='random-restaurant-custom-overlay'
+            class='random-restaurant-custom-overlay not-drag'
             src=${RESTAURANT_BADGE_IMAGE_FILE_PATH}
             style="
               width: 96px;
```

Comment:
```suggestion
            draggable='false'
            class='random-restaurant-custom-overlay'
            src=${RESTAURANT_BADGE_IMAGE_FILE_PATH}
            style="
              width: 96px;
```

p5: draggable이라는 img 태그의 속성을 false로 두면 드래그를 방지할 수 있는 것으로 알고 있습니다. 호환성 또한 잘 지원되는 것 같습니다!

reference
[MDN draggable](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable)
[호환성](https://caniuse.com/?search=draggable)

Reply:
오!! draggable은 처음알았어요! 더 깔끔하네요 !! 이거로 수정해두겠습니다 ~~